coderadar-core/src/main/java/io/reflectoring/coderadar/contributor/port/driver/GetContributorsForPathCommand.java

```diff
@@ -9,6 +9,6 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class GetContributorsForPathCommand {
-  @NotBlank private String path;
+  private String path;
```

Comment:
Why did you remove the `@NotBlank` annotation?

Reply:
It was a way to fix bug, but I've fixed it properly now.